docs/conf.py

```diff
@@ -25,7 +25,7 @@
 # -*- coding: utf-8 -*-
 #
 # Procrustes documentation build configuration file, created by
-# sphinx-quickstart on Wed Apr 11 19:35:53 2018.
+# sphinx-quickstart on Wed Apr 11 19:35:53 2020.
```

Comment:
I don't think this `2018` should be changed to `2020`.

Reply:
Problem fixed in b7e26dc. Thank for pointing this out.